dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJSPServer/server.xml

```diff
@@ -24,7 +23,4 @@
     <javaPermission className="java.security.AllPermission"  name="*" actions="*" />
     
     <mpMetrics authentication="false"/>
```

Comment:
No mpMetrics feature so this can be deleted, right?

Reply:
Fixed (and with the other files as well)